roles/postgres/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 download_folder: /opt
-postgres_repo_rpm_name: pgdg-centos96-9.6-3.noarch.rpm
-postgres_repo_download_url: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/{{postgres_repo_rpm_name}}
-postgres_repo_rpm_location: "{{download_folder}}/{{ postgres_repo_rpm_name}}"
+postgres_repo_rpm_name: pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
This will always fetch the default (which is much higher than 9.6). Can we put the default Postgres version
I am assuming, "postgres_repo_rpm_name" can be overridden by setup.yml?

Reply:
yes @angshu it will overridden by what we set as part of setup.yml